cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -460,8 +470,8 @@ private VoltageLevel newFictitiousVoltageLevel(Context context, LineContainerFic
                         context.nodeBreaker()
                                 ? TopologyKind.NODE_BREAKER
                                 : TopologyKind.BUS_BREAKER)
-                .setLowVoltageLimit(vlref != null ? vlref.getLowVoltageLimit() : 1.0)
```

Comment:
I am not sure that it is correct to set the nominal voltage to 1 here too... how do you retrieve the reference voltage level for a client connection in a middle of a line?

Reply:
We can't, this is why it is done a posteriori. I checked for every line configuration we have, they end up being set even if not done immediately.